### DISCUSSION TITLE
求助，请跟进，已付款，账号也有，就差一个激活码了

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#848] [Q&A]
@Mike777Khoo 我到现在都没收到激活码的邮件，请问什么时候会发送，是系统发送还是人工发送，就缺一个激活码的事情，给个时间节点吧，钱都给了结果就一直免费版250M一天，唉，说好的跟进呢？

**Comment by kidokogills** [comment]
8月29号付款，因为专业版过期后付款，旧账号已登出，无法自动续期
新账号说已更换 firekydchan@gmail.com ，

我能明白账号已经是专业版
但要正常使用，就需要一个激活码，登录账号
然而，从9月7号到现在，没收到任何激活码的邮件
我不明白，一个25位的字码为什么要这么长时间

**Comment by kidokogills** [comment]
说会跟进，但一直没有回复，也不知道有没有跟进
很想知道，什么时候可以处理，还能不能解决？激活码什么时候能发给我？ @Mike777Khoo

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/4bd5fcf1-bc8f-4cf5-aca8-da9f73fe6dfd)
以上的图示工作聊天记录. 早就帮您跟进好多次了
@kidokogills 您好. 上个星期五已经解决了啊. 技术部门给您更该去Gmail了. 说您可以登入了. 能收到激活码.

**Reply by kidokogills** [reply]
@Mike777Khoo 我的Gmail邮箱没有收到任何关于激活码的邮件

**Reply by kidokogills** [reply]
他只是说我可以用这个邮箱登录，但没给登录的激活码，我怎么登入？

**Reply by kidokogills** [reply]
> 他只是说我可以用这个邮箱登录，但没给登录的激活码，我怎么登入？

![image](https://github.com/user-attachments/assets/c0a189bc-79a5-41bd-b79d-02c3d317af47)
我的邮箱就这几个邮件，每一个是有关激活码的

**Reply by kidokogills** [reply]
@Mike777Khoo 如果他有发送激活码，请问有发送记录吗？

**Reply by kidokogills** [reply]
或者告诉我，这个界面，我下一步要做什么
![QQ截图20240910234113](https://github.com/user-attachments/assets/0c293d6d-caa3-4d6a-8a32-2669ae57e898)

**Reply by kidokogills** [reply]
@Mike777Khoo 麻烦你再跟进一下，我确实没收到激活码的邮件或者信息，如果可以的话，能不能再发一次到邮箱，既然是付费用户，激活码长什么样，我是有所有权的吧？

**Comment by MaxMKMax** [comment]
@kidokogills 
这个是技术部门给我的. 我是无权限给您弄激活码还是什么的. 您看可以吗?

767778

**Reply by kidokogills** [reply]
我该怎么回应这个......
是我中文不好，还是翻译错误，还是截图出错
我是专业版付费用户，需要的，是25位激活码，
前面的截图，显示需要25位激活码，邮箱的截图，显示的是没收到关于激活码的邮件
那么，按照正常逻辑，诉求重新发送一份激活码的邮件
给个6位的数字，我能有什么用？我又不是代理商
25跟6的区别，应该能分得清吧？要不要给截图团队看看，他们到底在发什么？
这是在敷衍我？一年的付费就强行吞了？还是说因为是中国用户，在搞歧视？


![QQ截图20240911152206](https://github.com/user-attachments/assets/3baf44e2-0228-4c24-b1cc-d0a55cb34eb7)


我需求就一个激活码，重新发送就能结案的事情
团队不是回复账号升级完成，就是更改账号，
我能理解账号已升级专业版，也明白可以收到激活码，但你们发出去了吗？
绕来绕去，就是不让我登录账号，是什么意思？

如果诉求还表述不清晰，那么我换个提问，

我的专业版账户被登出，激活码遗失，能否让团队重新发送一份激活码的邮件给我？
邮箱 firekydchan@gmail  ,

这个沟通、转告的权限应该有的吧？不然这个论坛到底在沟通什么，解决什么？

如果这个权限都没有，那么，请问，我遗失了激活码，有什么渠道可以解决这个问题？
目前就一个后台，就会自动回复，一个是这个用户论坛，还有什么渠道可以申诉？

**Comment by MaxMKMax** [comment]
@kidokogills 这个support的邮箱我是没权限管理的. 这个support是美国团队那里处理的.

**Reply by kidokogills** [reply]
@Mike777Khoo 这个邮箱的截图，是想跟你说，我没收到来自lantern团队的任何关于激活码的邮件

**Reply by kidokogills** [reply]
@Mike777Khoo 行了，我自行解决登录pro了，麻烦你了，感谢

**Reply by kidokogills** [reply]
@Mike777Khoo 然后新的问题来了，登录没多久，就变回免费版了。
无法通过 device linking code 登录 pro

ticket号码：76791388，logs ：

[lantern.log](https://github.com/user-attachments/files/16963812/lantern.log)

**Reply by kidokogills** [reply]
> @Mike777Khoo 然后新的问题来了，登录没多久，就变回免费版了。 无法通过 device linking code 登录 pro
> 
> ticket号码：76791388，logs ：
> 
> [lantern.log](https://github.com/user-attachments/files/16963812/lantern.log)

![微信截图_20240911205408](https://github.com/user-attachments/assets/596ff144-bb7d-4f1f-bc1b-61ed1e482c09)

**Reply by kidokogills** [reply]
授权设备有冲突，例如：电脑端或手机端，只使用其中一个，没有问题，但如果同时开，会把其中一个设备的号给撞出去，变成免费版

**Comment by MaxMKMax** [comment]
@kidokogills 超级抱歉的. 我也不知道为什么会一直这样. 我和技术部门反馈过过很多次了. 
谢谢您的log. 您遇到的问题我只能说声非常抱歉. 我看支付部门那里可以给您加时间来补偿您吗. 我尽我的能力帮您.

**Comment by MaxMKMax** [comment]
我会重视您遇到的问题. 一直帮您反馈到修复为止.

**Reply by kidokogills** [reply]
@Mike777Khoo 感谢，辛苦，在等，盼复

**Reply by kidokogills** [reply]
唉，今天连手机也登录不了了，验证码也发不到邮箱，全是免费版

**Reply by MaxMKMax** [reply]
@kidokogills 
您好. 这是您的验证码. 您看行吗. 谢谢您的耐心等待.
767778

**Reply by kidokogills** [reply]
@Mike777Khoo 試過了，不行，而且已經不只是郵箱收不到驗證碼的問題了，是輸入驗證碼后回到主頁，依然是免費版，PC端跟手機都是這種情況，顯示錯誤代碼500

**Reply by MaxMKMax** [reply]
@kidokogills 这个应该是个bug了. 您可以帮我提交各应用内报告吗? 然后给我那组LOG number好吗?

**Comment by MaxMKMax** [comment]
@kidokogills 我现在帮您反馈给技术部门. 看他们能怎么修复

**Reply by kidokogills** [reply]
@Mike777Khoo 所以现在是三个问题：
1、邮箱收不到授权设备验证码，即使是gmail邮箱；
2、在你这里获得验证码，输入后显示错误代码500；
3、如果输入验证码成功，刷新回到主页，依然显示免费版，无论是手机还是PC，都是以上情况


ticket：76975053
log 
[lantern.log](https://github.com/user-attachments/files/17018771/lantern.log)

如何登录成功，不变回免费版才是问题，也不知道是版本还是墙的问题，这样同类的问题我看论坛反映挺多的，希望能快些修复吧

**Reply by kidokogills** [reply]
> @Mike777Khoo 所以现在是三个问题： 1、邮箱收不到授权设备验证码，即使是gmail邮箱； 2、在你这里获得验证码，输入后显示错误代码500； 3、如果输入验证码成功，刷新回到主页，依然显示免费版，无论是手机还是PC，都是以上情况
> 
> ticket：76975053 log [lantern.log](https://github.com/user-attachments/files/17018771/lantern.log)
> 
> 如何登录成功，不变回免费版才是问题，也不知道是版本还是墙的问题，这样同类的问题我看论坛反映挺多的，希望能快些修复吧

tic : 76975612

**Reply by kidokogills** [reply]
> > @Mike777Khoo 所以现在是三个问题： 1、邮箱收不到授权设备验证码，即使是gmail邮箱； 2、在你这里获得验证码，输入后显示错误代码500； 3、如果输入验证码成功，刷新回到主页，依然显示免费版，无论是手机还是PC，都是以上情况
> > ticket：76975053 log [lantern.log](https://github.com/user-attachments/files/17018771/lantern.log)
> > 如何登录成功，不变回免费版才是问题，也不知道是版本还是墙的问题，这样同类的问题我看论坛反映挺多的，希望能快些修复吧
> 
> tic : 76975612

![QQ截图20240917033741](https://github.com/user-attachments/assets/16575768-3b7c-4476-bb0f-1078a4d61afb)

**Reply by kidokogills** [reply]
@Mike777Khoo 顺便问一句，邮箱收不到验证码，是不是把我的专业版账号给弄没了......

**Reply by MaxMKMax** [reply]
@kidokogills 账号不会没. 只是这Qiang太难翻了. 邮件发不过去.

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/a94b1efc-4bcb-4618-ba83-756109a6e364)

@kidokogills 已经帮您提交问题上去了. 现在等待回复. 我会尽快帮您催的啊.

**Comment by MaxMKMax** [comment]
@kidokogills 我这里凌晨三点了. 截图 3.06am 也帮您做事情啊. 谢谢您的体谅. 我会尽快帮您解决问题的. 请您放心!

**Reply by kidokogills** [reply]
@Mike777Khoo 感谢，我这也是，3：48 am，匹配美国时间

**Comment by greySkiesMarsh** [comment]
@kidokogills Hello, I'm sorry that you've had so much trouble with this. 

I've upgraded your account to Pro again, so please try to log in using the usual method. If that still doesn't work, try authorizing with the same email and use this code:
632DR-98HJG-KT74K-R277T-WRM48

**Reply by kidokogills** [reply]
@greySkiesMarsh @Mike777Khoo 
Thank you, now I have successfully logged in to pro. In order to verify whether the problem is solved, I will restart the program and computer in a few hours to see if the previous situation will occur

感谢，目前已成功登录pro，为了保险起见，验证问题是否得以解决，我会在数个小时后重启程序和电脑，看之前的情况会不会再次出现，辛苦了

**Reply by MaxMKMax** [reply]
@kidokogills 您好. 不客气呢. 谢谢您支持蓝灯. 能帮到您我很开心. 祝贺您中秋节快乐.
